Documentation/6.0.2/README.rst

```diff
@@ -0,0 +1,1518 @@
+    * The Layer 3 network has redundant VSDs.
+
+
+Red Hat
+
+    * During the certification process, the network should have an odd number of controllers so that the majority of the nodes are up if a node goes down.
+    * Go to https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/ for more Red Hat best practices.
+
+
+Integrating Nuage VSP with OpenStack Platform Director
+-------------------------------------------------------
+
+The integration includes the following steps:
+
+* Modifying the Overcloud qcow image (for example, overcloud-full.qcow2)
+
+    - The Nuage VRS and metadata agent configuration files need to be created and populated with the required parameters. To do this, add the puppet module (nuage-puppet-modules) to the Overcloud image with the other required Nuage RPMs.
+
+    - The typical OpenStack director deployment scenario assumes that all the packages are installed on the overcloud-full image. The Overcloud qcow image (for example, overcloud-full.qcow2) needs to be patched with the following RPMs:
+
+        - nuage-bgp
+        - nuage-metadata-agent
+        - nuage-openstack-heat
+        - nuage-openstack-horizon
+        - nuage-openstack-neutron
+        - nuage-openstack-neutronclient
+        - nuage-openvswitch (Nuage VRS)
+        - nuage-puppet-modules-5.1.0
+        - selinux-policy-nuage
+        - nuage-topology-collector
+
+    - Uninstall Open vSwitch (OVS).
+    - Install VRS (nuage-openvswitch).
+
+    - Use nuage-puppet-modules-5.1.0.x86_64.rpm and the nuage_overcloud_full_patch.py script to patch to the Overcloud qcow image, uninstall Open vSwitch (OVS), and install VRS.
+
+    - For AVRS integration, the overcloud-full image is also patched with following 6WIND and Nuage AVRS RPMs:
+
+        - 6windgate-dpdk
+        - 6windgate-dpdk-pmd-mellanox-rdma-core
+        - 6windgate-dpdk-pmd-virtio-host
+        - 6windgate-fp
+        - 6windgate-fpn-sdk-dpdk
+        - 6windgate-fp-ovs
+        - 6windgate-linux-fp-sync
+        - 6windgate-linux-fp-sync-fptun
+        - 6windgate-linux-fp-sync-ovs
+        - 6windgate-linux-fp-sync-vrf
+        - 6windgate-product-base
+        - 6windgate-tools-common-libs-daemonctl
+        - 6windgate-tools-common-libs-libconsole
+        - 6windgate-tools-common-libs-pyroute2
+        - 6wind-openstack-extensions
+        - dkms
+        - nuage-metadata-agent (6wind version)
+        - nuage-openvswitch (6wind version)
+        - python-pyelftools
+        - selinux-policy-nuage-avrs
+        - virtual-accelerator-base
+
+
+* Adding Nuage Heat Templates ( `nuage-tripleo-heat-templates <../../nuage-tripleo-heat-templates>`_  )
+
+    - Nuage provides heat templates & environment files to configure neutron on controller and nuage-openvswitch & nuage-metadata-agent on compute nodes and we configure VA on computeavrs nodes.
```

Comment:
Revise text:
Nuage provides Heat templates and environment files to configure Neutron on the Controller. Nuage also provides nuage-openvswitch and nuage-metadata-agent RPMs to configure on the Compute nodes. 

What do you mean by "we configure VA on computeavrs nodes" mean? Does the plugin configure VA on the computeavrs nodes?

Reply:
I think we can revise it to say:
"Nuage provides heat templates & environment files to configure neutron on controller and nuage-openvswitch & nuage-metadata-agent on compute nodes & Virtual-Accelerator on computeavrs nodes."

Reply:
Sounds good to me.

Reply:
done